h2o-py/tests/testdir_jira/pyunit_pubdev_5336.py

```diff
@@ -0,0 +1,42 @@
+import h2o
+import pandas as pd
+import numpy as np
+from tests import pyunit_utils
+
+
+def pubdev_5336():
+    data = pd.DataFrame({'Origin': ['SFO', 'SAN', 'SFO', 'NYC', np.nan],
+                         'Dest': ['SFO', 'SFO', 'SAN', 'SAN', np.nan]})
+    frame = h2o.H2OFrame(data)
+    frame['Origin'].asfactor()
+    frame['Dest'].asfactor()
+
+    # First column has one more categorical variable
+    assert frame['Origin'].nlevels() == [4]
+    assert frame['Origin'].levels() == [['NYC', 'SAN', 'SFO', 'nan']];
```

Comment:
This shows the test setup is actually incorrect - NA cannot be part of the domain.